src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js

```diff
@@ -65,6 +72,10 @@ export const WidgetEventEditPlain = ({
     if (error) {
         return error.errorComponent;
     }
+    const onHandleScheduleSave = (eventId: string, eventData: Object) => {
```

Comment:
Hey,
I suggest we avoid navigating from inside the widget and let the page deal with all the redirections. The same way it is currently done by other props of the `WidgetEventEdit` widget like [onGoBack](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.container.js#L77) and [onCancelEditEvent](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.container.js#L73).
Thanks!